nilearn/decoding/searchlight.py

```diff
@@ -166,10 +166,8 @@ def _group_iter_search_light(list_rows, estimator, X, y, groups,
     t0 = time.time()
     for i, row in enumerate(list_rows):
         kwargs = dict()
-        if not LooseVersion(sklearn.__version__) < LooseVersion('0.15'):
```

Comment:
Can you double check that this needs to be removed ?

Reply:
It will always evaluate to False, now that SKLearn is minimum v0.18. The logic will never be executed.

Reply:
>It will always evaluate to False

I could not understand how it would always evaluate to False.

Reply:
@KamalakerDadi Have Fixed it. Take another look, compare it with the original code (not the one here).